index.html

```diff
@@ -369,6 +371,139 @@ <h3>Disabled button</h3>
                     </code>
                 </pre>
             </div>
+
+            <!-- Image component starts -->
+            <div id="image">
+                <h2>Images</h2>
+                <p>Images can be responsive to fit the parent's width, and also can be customised to have rounded corners or as a circle</p>
+                <h3>Responsive Image</h3>
+                <p>You can add class <span class="doc-classname">img-responsive</span> to make your image fit the width of container. It's height will get adjusted by keeping the aspect ratio same.</p>
```

Comment:
Here class name is confusing if possible make a change.

Reply:
will change this